app/assets/stylesheets/nucore-variables.scss

```diff
@@ -20,3 +20,35 @@ $sidebarActiveBackgroundColor: $linkColor;
 $sidebarActiveBackgroundColorHighlight: darken($sidebarActiveBackgroundColor, 5%);
 
 $sidebarActiveColor: $white;
+
+/*
```

Comment:
Can we move this comment back to `nucore-overrides.scss`. The rest of this looks good.